tasks/storage_deal.go

```diff
@@ -145,19 +139,23 @@ func MakeStorageDeal(ctx context.Context, config NodeConfig, node api.FullNode,
 			storagemarket.StorageDealSlashed,
 			storagemarket.StorageDealRejecting,
 			storagemarket.StorageDealFailing,
-			storagemarket.StorageDealError,
+			storagemarket.StorageDealError:
+
+			logStages(info, log)
+			return errors.New("storage deal failed")
 
 			// deal is on chain, exit successfully
```

Comment:
unindent?

Reply:
Not sure what you mean, I usually leave that to `gofmt` / `goimports` - should I be using something else?

Reply:
it allows a comment in this line to either be indented with the body before, or with the case below. the comment logically refers to the comment below, so it should be unindented, like it. in some scenarios like this one, gofmt just follows your choice.